From d7eeac90fe02ccf5c5e4334e41ec75fc33803643 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Tue, 31 Mar 2020 16:12:28 +0200
Subject: [PATCH] meson.build: do not install python gi bindings

These are unnecessary for oe-core, and the install location
is incorrectly determined using native python.

Upstream-Status: Inappropriate [oe-core specific]
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>
---
 meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/meson.build b/meson.build
index a8b02b4..8ff737b 100644
--- a/meson.build
+++ b/meson.build
@@ -171,4 +171,3 @@ configure_file(
 )
 
 subdir('modulemd')
-subdir('bindings/python')
